### PR TITLE
data/vineetjaiss.json

### PR DESCRIPTION
**Pull request by vineetjaiss**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6877"></a>

### CODE REVIEW

File: VinJai.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Vineet Jaiswara ",
+  "type": "personal",
+  "bio": "Writer **Programmer**. Open SOurce Contributor",
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/thevinjai",
+      "icon": "FaTwitter"
+    }
```
Comment (khushalarora) [inline VinJai.json:10]: Please add a **'Comma'** at the end of this line :+1: 
```Suggestion
    },
```

File: VinJai.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    {
+      "name": "GitHub",
+      "url": "https://github.com/vineetjaiss/",
+      "icon": "FaGithub"
+    }
```
Comment (khushalarora) [inline VinJai.json:15]: Also, add a **'Comma'** here :+1: 
```Suggestion
    },
```

File: VinJai.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    {
+      "name":"LinkedIn",
+      "url":"https://linkedin.com/in/vineetjaiswara/",
+      "icon":"FaLinkedin"
```
Comment (khushalarora) [inline VinJai.json:19]: Also, add a **'Comma'** and **'bracket'** here :+1: 
```Suggestion
      "icon":"FaLinkedin"},
```

File: VinJai.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    {
+      "name":"Instagram",
+      "url":"https://instagram.com/vinjaiss"
+    }
+    }
```
Comment (khushalarora) [inline VinJai.json:24]: Delete this extra bracket :+1: 
```Suggestion
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hey @vineetjaiss, thank you for contributing :tada: 
Please rename your file to **`data/vineetjaiss.json`**.

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hey @vineetjaiss, thanks for the changes :+1: 
Instead of Title, you need to rename your file to **`data/vineetjaiss.json`**  [HERE](https://github.com/vineetjaiss/LinkFree/edit/vineetjaiss-patch-1/VinJai.json)

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Please read the review comments carefully. 

As suggested by Khushal, rename your file to exact `data/vineetjaiss.json` [here](https://github.com/vineetjaiss/LinkFree/edit/vineetjaiss-patch-1/vineetjaiss).

Change `vineetjaiss`  -->  `data/vineetjaiss.json`. Feel free to ask if you've any confusion.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**